### PR TITLE
- created env variables file and used variables from this file

### PR DESCRIPTION
**Pull request by kabiakou**
- replaced constants with variables from env file

### CODE REVIEW

File: .env
Comment (plutsenka) [inline .env:1]: .env file should never be stored on version control system, the better way is to store .env.example file with only var names without values

File: package.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
   "version": "0.0.1",
   "description": "Proxy server project",
   "main": "index.js",
-  "scripts": {  
-    "start": "node ./training/nodejs/js/index.js"
+  "scripts": {
+    "start": "node --env-file .env ./training/nodejs/js/index.js"
```
Comment (plutsenka) [inline package.json]: looks like i would be better to move entrypoint (index.js) to root directory and then just run 'node index.js' - dotenv will resolve .env file

File: training/nodejs/js/index.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 const https = require('https') 
 
-const FEED_URl = "https://api.nasa.gov/neo/rest/v1/feed"
+const API_KEY = process.env.NASA_API_KEY
+const FEED_URl = process.env.NASA_API_FEED_URL
 const START_DATE = "2024-02-26"
```
Comment (plutsenka) [inline training/nodejs/js/index.js]: it's a bad way to store dates as hardcoded string,  probably you will want to take a look at moment library

File: .env
Comment (kabiakou) [reply .env:1]: Fixed

File: package.json (hunk 1 above)
Comment (kabiakou) [reply package.json]: Moved index.js, but I left "--env-file .env" because do not use dotenv module

File: training/nodejs/js/index.js (hunk 2 above)
Comment (kabiakou) [reply training/nodejs/js/index.js]: Fixed

File: index.js [hunk 3]
```diff
@@ -1,20 +1,21 @@
...
+const https = require('https')
+const moment = require('moment')
 
-const FEED_URl = "https://api.nasa.gov/neo/rest/v1/feed"
-const START_DATE = "2024-02-26"
-const END_DATE = "2024-03-01"
-const API_KEY = "KhS7EQVPfk4NLgak8tr7LgNjVNbI0Feuj4q3cMC3"
+const API_KEY = process.env.NASA_API_KEY
+const FEED_URl = process.env.NASA_API_FEED_URL
+const START_DATE = moment('2024-02-26').format('YYYY-MM-DD')
```
Comment (plutsenka) [inline index.js:6]: moment() instead of moment('2024-02-26') will be definitely better here and below

Comment (kabiakou) [reply index.js:6]: Thanks. It seems like I will change time period "from Monday to Friday" to "from now to 5 days ago" :)

**Pull request merged**